### PR TITLE
Use root directory to remove binary files from repository

### LINKED ISSUE #4: Move source files to root directory
Currently, the source files reside in `thesis-style` while the generated zip resides in the root folder. However, GitHub allows "Download as zip" (right next to the clone URL). By moving the source files to the root directory, this "Download as zip" will be effectively the same as the binary zip file currently being uploaded. Lastly, the previous distributions can be available as GitHub release (which also supports adding files to a release). 

All in all, it should no longer be necessary to have binary files in this repository and is clearer what the latest version of the thesis template is.

### PR DESCRIPTION
**Pull request by TimvdLippe**
I went ahead to implement #4 as it was quite an easy change and now you can actually see the "Download as zip" in action. If you visit https://github.com/SERG-Delft/thesis-template/tree/use-root-directory and click the green button "Clone or download" and then click "Download ZIP" you get the same ZIP as you previously had, but now versioned by commit.

Moreover, I added Travis support as my ShareLatex build was actually failing on a fresh import. Looking more closely at the error, it seems that ShareLatex actually is incapable of importing `*.ps` images. Nonetheless, any Latex errors will now be caught on https://travis-ci.org/SERG-Delft/thesis-template instead. We could trim down the installation time by not installing `texlive-full`, but that would be a future enhancement. (There is no clear documentation on what Latex dependencies are required for this project)

Closes #4

### CODE REVIEW

**Comment by Keraito** [comment]
This, especially the `README.ME`, made it really clear for me how to set up the template on ShareLatex. Documentation-wise certainly an improvement 🎉.

File: README.md
Comment (Inventitech) [inline README.md]: Actually, Debian-based systems (apt!)

File: thesis-style/Makefile
Comment (Inventitech) [inline thesis-style/Makefile]: why remove this entirely?

**Review by Inventitech — CHANGES_REQUESTED** [review]

File: Makefile
Comment (Inventitech) [inline Makefile:1]: How about defining a constant to hold acroread, and having that point to evince by default (imho, acroread support was dropped > 2 years ago)?

What about using mklatex to crate the pdf?

File: README.md
Comment (Inventitech) [inline README.md:1]: What about a link to https://github.com/dspinellis/latex-advice? :-)

File: .travis.yml
Comment (Inventitech) [inline .travis.yml:1]: nice :-)

**Review by Inventitech — CHANGES_REQUESTED** [review]

**Comment by Inventitech** [comment]
If you want to make the Travis CI builds faster, have a look at https://github.com/harshjv/travis-ci-latex-pdf/blob/master/.travis.yml, which has a widely working but fairly minimal set of dependencies to compile LaTeX. The other thing to copy from there might be the interactive mode.

File: Makefile
Comment (TimvdLippe) [reply Makefile:1]: I see no reference to neither `acroread` nor `evince`. Could you explain what they should be used for?

`mklatex` seems like a good choice :+1:

File: Makefile
Comment (Inventitech) [inline Makefile]: reference to acroread?
Personally, I would just remove the view target, actually.

Comment (TimvdLippe) [reply Makefile]: Oh I completely missed that. Yes that should have been deleted.

**Comment by TimvdLippe** [comment]
All feedback processed and build green again :tada:

**Comment by Inventitech** [comment]
You made @avandeursen a reviewer on this. Any particular reason for it? If you want to wait for what he has to say, that's fine. For me, it is good to go now.

**Comment by TimvdLippe** [comment]
I initially thought he was the maintainer of this repository, as he was referenced in the template as well. He is probably busy, so I think this is good to go without his explicit approval :smile:

**Comment by Inventitech** [comment]
Really nice work, Tim!

**Pull request merged**
